core/javacc.gradle

```diff
@@ -2,7 +2,7 @@
 task downloadJavacc {
     ext.file = "javacc-5.0.tar.gz"
     ext.dest = file("$buildDir/javacc/$file")
-    ext.url = "https://javacc.org/downloads/$file"
+    ext.url = "http://download.openpkg.org/components/cache/javacc/$file"
```

Comment:
Is there a reason that this was updated for using openjdk8? If we do this we'd like this to be a URL that works over `https`.

Reply:
No reason with openjdk8. 
It is another issue with the gradle build (last successful buld was 1 year ago). 
I didn't found an URL over `https` which serve `javacc-5.0.tar.gz` (i don"t know why this version does not exist on the new javacc download page https://github.com/javacc/javacc/releases)

Reply:
Let's split this up into another PR. The openjdk8 is a no brainer. It appears due to this not being available over https, javacc may take a bit more work.